### ISSUE TITLE
Request: keep current zoom settings when changing phase

### ISSUE THREAD

**Issue by GijsTimmers** [zond/android-diplicity#85]
As of now, when changing phase, the map zooms out to default zoom settings. It would be nice if the current zoom settings would be applied after changing phase.

**Comment by zond** [comment]
Yes! I would love that! But I don't know how to either get or set the zoom
settings from web views :/
